draft-ietf-ppm-dap.md

```diff
@@ -1049,7 +1058,7 @@ struct {
       number generator.
 
     * `time` is the time at which the report was generated. The Client SHOULD
-      round this value down to the nearest multiple of the task's
+      truncate this value down to the nearest multiple of the task's
```

Comment:
I don't object to using the verb "truncate" here (is the precedent [Go's `time.Truncate`](https://pkg.go.dev/time#Time.Truncate)?), but "truncate [...] down" is awkward. There's no such thing as "truncating up" so why say "truncate down"?
```suggestion
      truncate this value to the nearest multiple of the task's
```